dGame/dComponents/PetComponent.cpp

```diff
@@ -70,8 +70,8 @@ std::map<LOT, int32_t> PetComponent::petFlags = {
 		{ 13067, 838 }, // Skeleton dragon
 };
 
-PetComponent::PetComponent(Entity* parentEntity, uint32_t componentId) : Component{ parentEntity } {
-	m_PetInfo = CDClientManager::GetTable<CDPetComponentTable>()->GetByID(componentId); // TODO: Make reference when safe
+PetComponent::PetComponent(Entity* parentEntity, uint32_t componentId) : Component{ parentEntity },
+	m_PetInfo{ CDClientManager::GetTable<CDPetComponentTable>()->GetByID(componentId) } {
```

Comment:
Please pull out to another PR.  On this note, the object in question was not updated to be a reference which should be done as well.

Reply:
I no longer want to include a reference member due to it inhibiting move constructors and reducing data locality.